tests/test_schema.yaml

```diff
@@ -370,6 +370,42 @@ policies:
   principal: admin
   resource:
     path: /v2.0/responder.*
+- action: '*'
+  condition:
+  - is_owner
+  effect: allow
+  id: allow_attacher
+  principal: Member
+  resource:
+    path: /v2.0/attacher.*
+- action: '*'
+  condition:
+  - is_owner
+  effect: allow
+  id: allow_attach_target
+  principal: Member
+  resource:
+    path: /v2.0/attach_target.*
+- action: 'attach'
+  id: attach_if_accessible
+  effect: allow
+  principal: Member
+  source:
+    resource_schema_id: attacher
```

Comment:
source schema_id can be specified by resource.path (same policy can be applied to multiple types), so I do not think we need to add this property.

I will not stick to above idea, but it might be worth to consider.

Reply:
I removed this field.